components/sidebar-navigation/Component.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect } from "react";
+import { cnb } from "cnbuilder";
+
+// import specific templates for the component
+import BurgerBar from "../../packages/SVG-library/BurgerBar";
+import Close from "../../packages/SVG-library/Close";
+import LinkItem from "./Components/LinkItem";
+
+/**
+ * Base component
+ *
+ * @param {string} title The component title
+ * ... any other options needed
+ * @returns {JSX.Element}
+ * @constructor
+ */
+
+export default function sidebarNavigation({
+  asset_assetid,
+  asset_short_name,
+  asset_url,
+  active,
+  menu,
+}) {
+  const [navOpen, setNavOpen] = useState(false);
+  const [showMobileButton, setShowMobileButton] = useState(false);
+
+  useEffect(() => {
+    if (window.innerWidth > 991) {
+      setShowMobileButton(false);
+      setNavOpen(true);
+    } else if (window.innerWidth < 991) {
+      setShowMobileButton(true);
+      setNavOpen(false);
+    }
+  }, []);
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerWidth > 991) {
+        setShowMobileButton(false);
+        setNavOpen(true);
+      } else if (window.innerWidth < 991) {
+        setShowMobileButton(true);
+        setNavOpen(false);
+      }
+    };
+
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
+  return (
+    <>
+      {showMobileButton && (
+        <button
+          data-role="sidebar-navigation-toggle"
+          aria-controls="sidebar-navigation"
+          aria-label="sidebar-navigation"
+          aria-expanded={!!navOpen}
+          type="button"
+          onClick={() => setNavOpen(!navOpen)}
+          className={cnb(
+            "lg:su-hidden su-flex su-items-center su-w-full su-h-[5.6rem] su-p-10 su-text-left su-font-semibold su-border-2 su-border-digital-red su-text-digital-red dark:su-border-dark-mode-red dark:su-text-dark-mode-red",
+            navOpen &&
+              "su-bg-digital-red su-text-white dark:su-bg-dark-mode-red dark:aria-expanded:su-text-black-true"
+          )}
+        >
+          <span className="su-flex-auto">
+            {navOpen ? "Close" : "Section menu"}
+          </span>
+          {navOpen ? <Close /> : <BurgerBar />}
+        </button>
+      )}
+      {navOpen && (
+        <nav className="" data-role="sidebar-navigation-wrapper">
```

Comment:
I believe this `nav` would need an `aria-label` to distinguish itself from other nav landmarks on the site. SODA suggested adding aria-labels like "sidebar menu". The screen reader will add "navigation" to the end so it will read it as "sidebar menu navigation" so no need to include the word "navigation" in the aria-label 😄

Reply:
Since your mobile button has `aria-controls="sidebar-navigation"`, I believe this would mean that the nav here would need an `id="sidebar-navigation"` However, the aria-controls attribute might be optional here. I do suggest going to SODA office hour for a quick check for this component.